build_config/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 - Add `TargetBuilderConfigDefaults` class for Builder authors to provide default
   configuration.
 - Add `InputSet` and change `sources` and `generate_for` to use it.
+- Remove `BuildTarget.isDefault`. The default will be determined by the target
```

Comment:
should we mention it was removed from the config as well?

Reply:
Done